packages/core/src/utils/MessageValidator.ts

```diff
@@ -11,4 +14,24 @@ export class MessageValidator {
   public static validate<T extends object>(classInstance: T) {
     return validateOrReject(classInstance)
   }
+  // eslint-disable-next-line @typescript-eslint/ban-types
+  public static validateSync<T extends object>(
+    // eslint-disable-next-line @typescript-eslint/ban-types
+    classInstance: T & {},
```

Comment:
```suggestion
    classInstance: T,
```

I believe this would suffice.

Reply:
Maybe there is a good way to keep it `T` only? Added & {} because that is the type returned by `plainToInstance`